blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpEventsQueryHandler.java

```diff
@@ -29,47 +30,37 @@ public HttpEventsQueryHandler(EventsIO searchIO) {
     @Override
     public void handle(ChannelHandlerContext ctx, HttpRequest request) {
 
-        if(request.getMethod().toString().equalsIgnoreCase("GET")) {
```

Comment:
Why was this check for GET removed? Don't we want to only serve GET requests for the URI /v2.0/tenantId/events/getEvents ?